cmake/dependencies.cmake

```diff
@@ -281,6 +288,11 @@ if(ENABLE_OPENCV)
             elseif((LINUX_OS_NAME STREQUAL "Ubuntu 20.04" OR LINUX_OS_NAME STREQUAL "LinuxMint 20.1") AND X86_64)
                 set(OPENCV_SUFFIX "ubuntu20")
                 set(OPENCV_HASH "2fe7bbc40e1186eb8d099822038cae2821abf617ac7a16fadf98f377c723e268")
+            elseif(LINUX_OS_NAME STREQUAL "CHROMIUMOS" AND X86_64)
+                find_package(OpenCV)
+                if (NOT OpenCV_FOUND)
```

Comment:
using this command you will find system OpenCV, which is Linux x86 one. Why not to use OpenCV corresponding to [CMKAE_HOST_SYSTEM](https://cmake.org/cmake/help/v3.0/variable/CMAKE_HOST_SYSTEM.html)?